README.md

```diff
@@ -392,11 +392,6 @@ while (iter.hasNext()) {
 }
 ```
 
-Note: in the above example, the following will throw a `ReadOnlyBufferException` because the buffer mode is read-only:
```

Comment:
I looked on the original README and didn't understand from where ReadOnlyBufferException should come... Where is the write attempt? Can you tell me?

Reply:
Below (line 397), we try to use `putInt()` on a read-only buffer.
This is no longer will issue an exception because the unsafe API will always produce a writable buffer.